### PR TITLE
Add `!handbook` command to  search the TypeScript Handbook

### PR DESCRIPTION
**Pull request by tjjfvi**
image

### CODE REVIEW

File: src/modules/handbook.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+		]);
+		const hit = data.results[0].hits[0];
+		if (!hit)
+			return await sendWithMessageOwnership(
+				msg,
+				':x: No results found for that query',
+			);
+		const hierarchyParts = [0, 1, 2, 3, 4, 5, 6]
+			.map(i => hit.hierarchy[`lvl${i}`])
+			.filter(x => x);
```
Comment (robertt) [inline src/modules/handbook.ts:58]: Is this needed?

**Review by robertt — APPROVED** [review]
LGTM, other then one minor thing

File: src/modules/handbook.ts (hunk 1 above)
Comment (tjjfvi) [reply src/modules/handbook.ts:58]: What part are you referring to? The structure of the object is:
```json
{
  "lvl0": "Documentation",
  "lvl0": "Documentation",
  "lvl0": "Documentation",
  "lvl0": "Documentation",

Comment (robertt) [reply src/modules/handbook.ts:58]: Sorry, was referring to the .filter(x => x)

Comment (robertt) [reply src/modules/handbook.ts:58]: Ahh my bad, sorry, way too tired to be doing code reviews 😅

Comment (tjjfvi) [reply src/modules/handbook.ts:58]: I can remove the `.map` and make it `[hit.hierarchy.hit0, hit.hierarchy.hit1, ...]`, but this seemed DRYer to me.

**Comment by robertt** [comment]
@tjjfvi Is this good to go on your end?

**Comment by tjjfvi** [comment]
@robertt Yup

**Pull request merged**